README.md

```diff
@@ -28,19 +28,30 @@ Contributions are very welcome.
 
 Suggested flow:
 
- - Open an issue to discuss the fix / improvement. Might not be needed for trivial things.
- - Fork the repo and create a feature/bug-fix branch from `dev` branch. For trivial changes you might want to work directly on forked `dev` branch.
+ - Open an issue to discuss the fix / improvement. Might not be needed for 
+ trivial things.
+ - Fork the repo and create a feature/bug-fix branch from `dev` branch. 
+ For trivial changes you might want to work directly on forked `dev` branch.
  - Send a PR to `dev` branch
 
+You will find the full description on top of the [issues](issues). There are 
+also existing templates that will be used automatically when opening a new 
+issue or pull request.
+
 # Security Issues
 
-If you have identified a security issue, please email **giuseppe.mazzapica [at] gmail.com** and do not
+If you have identified a security issue, please email 
```

Comment:
That line-selection happen regardless - it just seems a lot of extra work to break up (and to maintain) lines at a certain point, on the off chance someone wants to reference a change in docs. Besides, a diff will highlight the changed words anyway, even in an extremely long paragraph that automatically wraps across multiple lines.